packages/ppx_distr_guards/ppx_distr_guards.0.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Extension to distribute guards over or-patterns"
+description:
+  "`function%distr A x, _ | _, A x when p x -> e` will result in `function A x, _ when p x -> e | _, A x when p x -> e`"
+maintainer: ["Ralf Vogler <ralf.vogler@gmail.com>"]
+authors: ["Ralf Vogler <ralf.vogler@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/vogler/ppx_distr_guards"
+doc: "https://vogler.github.io/ppx_distr_guards"
+bug-reports: "https://github.com/vogler/ppx_distr_guards/issues"
+depends: [
+  "dune" {>= "2.7" & >= "2.3.0"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```

Reply:
This got generated from the [dune-project](https://github.com/vogler/ppx_distr_guards/blob/0ec3a096baa6f147c8981eb1f2d11fb72f865f2a/dune-project#L17) file.
If I adjust the constraint there to `(dune (>= 2.7))` the generated opam constraint is `"dune" {>= "2.7" & >= "2.7"}`. I assume it adds `2.7` from `(lang dune 2.7)`.
Should this be reported somewhere or do you just manually adjust or ignore it (or leave out the dune constraint)?

Reply:
This is a dune issue. See https://github.com/ocaml/dune/pull/3705